src/content/reference/manifest.mdx

```diff
@@ -72,6 +78,8 @@ You can build all these images by running `okteto build`, or `okteto build xxx`
 
 > if a `<service>` name includes the symbol `-`, in the corresponding `OKTETO_BUILD_<service>` environment variables, the symbol `-` will be replaced by `_`.  For example, if `<service>` is `name-with-hyphen`, the environment variable names will start with `$OKTETO_BUILD_NAME_WITH_HYPHEN_`.
 
+> Okteto will automatically add all the build environment variables from previous builds as build arguments. To refer to them, remember to add the `ARG` instruction on your Dockerfile.
```

Comment:
What do we mean by "previous builds"?

Reply:
The builds of the services that has been already build (the ones that are under `depends_on` section)